scripts/world_bank/datasets/datasets.py

```diff
@@ -0,0 +1,214 @@
+import requests
```

Comment:
Can you add the copyrigth header
Pls also add a file level block comment/readme on how to run the script.

Reply:
Done.